src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -5,6 +5,22 @@
       "toolInfo": {
         "name": "AndroidStudio"
       },
+      "runInfo": {
+        "fileInfo": {
+          "app/src/androidTest/java/microsoft/androidstudioformatexample/ApplicationTest.java": [
+            {
+              "uri": "app/src/androidTest/java/microsoft/androidstudioformatexample/ApplicationTest.java",
+              "mimeType": "text/x-java-source"
+            }
+          ],
+          "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java": [
+            {
+              "uri": "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java",
+              "mimeType": "text/x-java-source"
+            }
+          ]
+        }
+      },
       "results": [
```

Comment:
All the test data file changes look like this. `fileInfo` appears in `runInfo` (and so `runInfo` appears in files where it didn't before)...